neurodamus/neuromodulation_manager.py

```diff
@@ -74,7 +75,9 @@ def finalize(
             syn_params = self._synapse_params[syn_i]
             # We need to get all connections since we dont know the sgid
             # TODO: improve this by extracting all the relative distances only once
-            base_conns = base_manager.get_connections(self.tgid)
+            base_conns = chain(
+                *[base_manager.get_connections(self.tgid) for base_manager in base_managers]
+            )
```

Comment:
```suggestion
base_conns = chain.from_iterable(
    base_manager.get_connections(self.tgid) for base_manager in base_managers
)
```

Reply:
... if you want to flatten this is more efficient

Reply:
Good point, done!